### PR TITLE
Adding hits_time_in_millis and misses_time_in_millis to enrich cache stats

### PR DESCRIPTION
**Pull request by masseyke**
This adds hits_time_in_millis and misses_time_in_millis to the enrich cache stats returned by `/_enrich/_stats` to bring it inline with geoip cache stats (#107334).

### CODE REVIEW

File: x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/enrich/action/EnrichStatsAction.java [hunk 1]
```diff
@@ -200,6 +214,8 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                 builder.field("hits", hits);
                 builder.field("misses", misses);
                 builder.field("evictions", evictions);
+                builder.field("hits_time_in_millis", hitsTimeInMillis);
+                builder.field("misses_time_in_millis", missesTimeInMillis);
```
Comment (dakrone) [inline x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/enrich/action/EnrichStatsAction.java:218]: Random drive-by, but can we make these use `builder.humanReadableField("hits_time_in_millis", "hits_time", new TimeValue(hitsTimeInMillis))` so that the `?human` flag can be used for the stats? Sometimes it's nice to be able to see "oh 14.2 minutes of hit time" instead of doing the millisecond-to-human conversion.

**Review by joegallo — APPROVED** [review]
